### PR TITLE
[Editor] fbx/preview's inspector choice first animation

### PR DESCRIPTION
**Pull request by chichinohaha**
use the clip info of the first animation whether or not the boardcast is received

Re: cocos-creator/3d-tasks#

### CODE REVIEW

File: editor/inspector/assets/fbx/preview.js [hunk 1]
```diff
@@ -236,7 +245,41 @@ exports.close = function () {
 };
 
 exports.methods = {
-    async refreshPreview() {
+    getCurClipInfo () {
```
Comment (arsen2010) [inline editor/inspector/assets/fbx/preview.js]: 这个也可以复用 Animation 模块里的吧？

Comment (chichinohaha) [reply editor/inspector/assets/fbx/preview.js]: now  it's reuse

**Review by arsen2010 — APPROVED** [review]

File: editor/inspector/assets/fbx/animation.js [hunk 2]
```diff
@@ -303,14 +303,14 @@ exports.$ = {
 const Elements = {
     // infos put first
     infos: {
-        ready() {
+        ready () {
```
Comment (VisualSJ) [inline editor/inspector/assets/fbx/animation.js:306]: Did you  need Spaces after function names?
If necessary, please format all the code correctly in the standalone PR.

**Review by VisualSJ — CHANGES_REQUESTED** [review]

File: editor/inspector/assets/fbx/animation.js (hunk 2 above)
Comment (chichinohaha) [reply editor/inspector/assets/fbx/animation.js:306]: yes
latter

**Pull request merged**